src/sentry/integrations/github/webhook.py

```diff
@@ -664,10 +666,12 @@ def handle(self, request: HttpRequest) -> HttpResponse:
             return HttpResponse(status=204)
 
         try:
-            method, signature = request.META["HTTP_X_HUB_SIGNATURE"].split("=", 1)
-        except (KeyError, IndexError):
```

Comment:
isn't `IndexError` a valid error to catch in case `HTTP_X_HUB_SIGNATURE` does not exist in `request.META`?

Reply:
jk i it should be KeyError